.markdownlint.json

```diff
@@ -1,6 +1,6 @@
 {
-    "default": false,
-    "MD009": {
-        "br_spaces": 0
-    }
+    "MD013": false,
+    "MD024": false,
+    "MD004": false,
+    "MD034": false
```

Comment:
Nit, MD004 should come before the other MDs, also want to make sure that this is the correct rule since it is between 24 and 34.